vscode-web/main.tf

```
@@ -53,30 +53,48 @@ variable "log_path" {
   default     = "/tmp/vscode-web.log"
 }
 
-variable "install_dir" {
+variable "install_prefix" {
   type        = string
-  description = "The directory to install VS Code CLI"
-  default     = "/tmp/vscode-cli"
+  description = "The prefix to install vscode-web to."
+  default     = "/tmp/vscode-server"
```

```suggestion
  default     = "/tmp/vscode-web"
```

We should probably attempt naming consistency between our examples and default values.